lib/util.js

```diff
@@ -64,9 +72,15 @@ function universalHapiPlugin(registers, pkg) {
 
 module.exports = {
   hapiMajor,
-  isHapi17,
+  isHapi17OrUp: isHapi17,
   isHapi18OrUp,
   universalHapiPlugin,
   _testSetHapi17,
-  _testSetHapi18
+  _testSetHapi18,
+  isHapi17
 };
+
+Object.defineProperty(module.exports, "hapiVersion", {
+  get: () => hapiMajor(),
+  set: v => (HAPI_MAJOR_VERSION = v)
```

Comment:
Is there any value in throwing a "+" in front of the version here in case the consumer inadvertently passes a string?

Reply:
typescript! :)

good point, but don't see a need for it because then it raises the question should we deal with the user passing in other invalid values?  This is mainly for testing so not too concerned about it.